templates/wechatgame/game.ejs

```diff
@@ -64,7 +65,12 @@ function onApplicationCreated(application) {
         require('./engine-adapter');
         return application.init(cc);
     }).then(() => {
-        return firstScreen.end().then(() => application.start());
+        return firstScreen.setProgress(0.8).then(() => {
+            cc.director.once(cc.Director.EVENT_BEGIN_FRAME, () => {
```

Comment:
Call first.end() to stop firstScreen rendering until the engine renders to avoid black screen